hassio/src/dialogs/repositories/dialog-hassio-repositories.ts

```diff
@@ -233,6 +233,7 @@ class HassioRepositoriesDialog extends LitElement {
       this._repositories = await fetchStoreRepositories(this.hass);
 
       fireEvent(this, "supervisor-collection-refresh", { collection: "addon" });
+      fireEvent(this, "supervisor-collection-refresh", { collection: "store" });
```

Comment:
`_loadData` is called in `showDialog` and in `_removeRepository`. We don't need to refresh the store in these cases. I think it makes more sense to only refresh it in `_addRepository` directly

Reply:
Agreed for `showDialog`, but for `_removeRepository`, wouldn't we want that repository/its addons not to show up upon removal?